config/sync/core.extension.yml

```diff
@@ -44,6 +47,7 @@ module:
   itk_azure_video: 0
   itk_cookie_message: 0
   itk_media_entity: 0
+  itk_pretix: 0
```

Comment:
Are we missing `itk-dev/itk_pretix` in `composer.json`?

Reply:
Added to composer.json and composer.lock